src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -176,17 +178,19 @@ private static void SaveFileConfig(FileConfig fileConfig, ILogger logger)
                 {
                     using (var binaryWriter = new BinaryWriter(writer.BaseStream))
                     {
-                        Protocol.Write(binaryWriter, request);
+                        // TODO: need to handle different types of IRequest
+                        var vcxProjRequest = (Request)request;
+                        Protocol.Write(binaryWriter, vcxProjRequest);
```

Comment:
I've changed the methods related to calling the subprocess and handling the response to use the new interface.
This is the only place where we need to downcast.